core/base/src/main/java/alluxio/exception/ExceptionMessage.java

```diff
@@ -204,6 +206,10 @@
   INCOMPATIBLE_VERSION("{0} client version {1} is not compatible with server version {2}"),
 
   // configuration
+  UNABLE_TO_DETERMINE_MASTER_HOSTNAME("Cannot run {0}; Unable to determine {1} address. Please "
```

Comment:
can we add a test somewhere that generates this exception message, and verifies the `String` result instead of testing the exception message against itself? i.e. `assertEquals(e.getMessage(), "...")` instead of `assertEquals(e.getMessage(), ExceptionMessage.UNABLE_TO_DETERMINE_MASTER_HOSTNAME.getMessage(...))`

Reply:
Added